aws-core/src/main/java/software/amazon/awssdk/awscore/config/AwsAdvancedClientOption.java

```diff
@@ -46,6 +46,8 @@
     public static final AwsAdvancedClientOption<Boolean> ENABLE_DEFAULT_REGION_DETECTION =
         new AwsAdvancedClientOption<>(Boolean.class);
 
+    public static final AwsAdvancedClientOption<String> SIGNING_NAME = new AwsAdvancedClientOption<>(String.class);
```

Comment:
Is there a more verbose name for this? SERVICE_SIGNING_NAME?

Reply:
It might make it more clear. Renamed.